include/TuningsImpl.h

```diff
@@ -29,7 +29,6 @@ inline std::istream &getlineEndingIndependent(std::istream &is, std::string &t)
 {
     t.clear();
 
```

Comment:
Ooooh removing the sentry is not correct. Thats a guard class which sets internal bits

Reply:
@baconpaul 
OK, so perhaps the fix should be to test the sentry object is valid after construction?
That way it wouldn't be flagged as unused by an analyser.

Reply:
@baconpaul See what you think of the latest commit.